files/ansible/tasks/call_harness.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Get CAP_SYS_ADMIN
+  shell: cat /proc/sys/kernel/perf_event_paranoid
+  register: cap_sys_admin
+- debug: var=cap_sys_admin
+
+- name: Set CAP_SYS_ADMIN to 2
+  shell: echo "2" | tee /proc/sys/kernel/perf_event_paranoid
+  when: cap_sys_admin.stdout != '2'
+
+- name: Call the benchmark_harness
+  shell: './benchmark_harness/benchmark_controller.py {{benchmark|default("") }} "--machine_type={{machine_type|default("") }}" "--toolchain={{compiler|default("") }}" "--compiler-flags={{compiler_flags|default("") }}" "--linker-flags={{linker_flags|default("") }}" "--run-flags={{run_flags|default("") }}" "--size={{size|default("") }}" "--iterations={{iterations|default("") }}" "--benchmark-root="results"" {{ harness_options|default("") }}'
+  register: benchmark_call_output
+- debug: var=benchmark_call_output.stderr_lines
```

Comment:
This will always be empty if we don't pass -v, right? I think we should always pass -at least- -v, but possible -vv.

Reply:
ahah see your comment : https://github.com/Linaro/benchmark_harness/issues/26
I only need to check that the -v or even --v output is safe enough for Jenkins Console log. Or should we consider that Jenkins job's access control is enough ?

Reply:
-v and -vv outputs are safe in that they don't print out/err files to the tty, but to files directly.